### PR TITLE
Disable Create button in template after initial click

### PR DESCRIPTION
**Pull request by hiba-aldalaty**
## Hey, I just made a Pull Request!

Issue: https://github.com/backstage/backstage/issues/7045

This implementation adds a 'disabled' attribute to the create button in a template. This is set to true once the customer clicks the 'create' button.

This would help prevent confusion and accidental creation of multiple applications when there is slowness in starting template execution after 'create' is clicked. E.g., if customer clicks 'create' and template execution takes a few seconds to start, the customer may try to click again. 

Now,
Before click:
image

After click:
image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by freben — COMMENTED** [review]
Should we perhaps disable all of the buttons?

**Comment by hiba-aldalaty** [comment]
> Should we perhaps disable all of the buttons?

Good catch - customers shouldn't be able to click 'Back' or 'Reset' once execution starts either.

I'll push a commit shortly.

File: .changeset/sixty-carrots-marry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': patch
+---
+
+Disable 'Create' button in template after initial click to avoid confusion when there is slowness in template execution
```
Comment (freben) [inline .changeset/sixty-carrots-marry.md:5]: now accidentally we have two files here :) but apart from that it looks great!

Comment (hiba-aldalaty) [reply .changeset/sixty-carrots-marry.md]: Removed this one

**Review by freben — APPROVED** [review]

**Pull request merged**